app/models/miq_bulk_import.rb

```diff
@@ -52,7 +52,7 @@ def self.upload(fd, tags, keys)
   def self.find_entry_by_keys(klass, keys)
     keys2array = keys
     primary_key, primary_key_value = keys2array.shift
-    result = klass.where("LOWER(#{primary_key}) LIKE '#{primary_key_value.downcase}'")
+    result = klass.where(klass.arel_attribute(primary_key).lower.eq(primary_key_value.downcase))
```

Comment:
I'm curious what the "primary_key" is to tell you the truth.
I'm guessing it is "name" or something similar. not the actual klass.primary_key